.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Fix
+about: Did you fix something that did not work as expected?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
+-->
+
+# 🐛 bug fix
+
+<!--- Provide the details of the pull request here -->
+
+## 🤔 What was not working?
+
+<!--- Tell us what should happen -->
+
+## 💁 Your Solution
+
+<!--- Describe how you fixed the bug -->
+
+## 🔦 Tests
+
+<!--- Providing tests is not always necessary but helps us test for future bugs -->
+
+## 🌍 Your Environment
+
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+
+| Software         | Version(s) |
+| ---------------- | ---------- |
+| Parcel           |
+| Node             |
+| npm/Yarn         |
+| Operating System |
+
+<!-- Love parcel? Please consider supporting our collective:
```

Comment:
Probably also a good idea to add a checklist to the end of the PR to make sure they pass all the necessary requirements for a PR, something like this:

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions
- [ ] Included links to related issues/PRs